examples/Example8_SendReceiveBinary/helper_module.cpp

```diff
@@ -0,0 +1,94 @@
+#include "helper.h"
+#include <Notecard.h>
+
+struct context
+{
+	size_t buf_size;
```

Comment:
Prefer spaces over tabs. No technical reason, it's just what we do in the rest of the repo. 4 spaces to indent.

Reply:
Oddly enough I can't figure this one out. Other .cpp files in VS Code save with spaces, but when I change this one and save it, VS Code replaces the spaces with tabs. I don't use prettier either.

Reply:
Weird. NBD.